src/crypto/openssl/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ set(openssl_crypto_plugin_srcs
   openssl_crypto_plugin.cc)
 
 add_library(ceph_crypto_openssl SHARED ${openssl_crypto_plugin_srcs})
-target_link_libraries(ceph_crypto_openssl PRIVATE crypto)
+target_link_libraries(ceph_crypto_openssl
+    PRIVATE OpenSSL::Crypto
```

Comment:
could you help me understand this change?

Reply:
Cmake was failing to retrieve the openssl ``crypto`` library, unless explicitly using ``OpenSSL::Crypto``. I can check if this still needed with recent Cmake versions.

Reply:
Looks like this change is no longer needed, at least using Cmake 3.16.3. I'm dropping it right away.

Reply:
@petrutlucian94 i double checked `openssl_crypto_accel.cc`, it seems that it is using functions exposed by OpenSSL, i think we should link it against `ceph_crypto_openssl`.